### DISCUSSION TITLE
o algoritmo nao esta funcionando

### DISCUSSION THREAD

**Discussion by rebecsvvv** [dgadelha/Portugol-Webstudio#189] [Dúvidas / Perguntas] [answered]
real valorporhora,horastrabalhadas, salariototal

    escreva("Digite o valor que o funcionário ganha por hora: ")
    leia(valorporhora)
    escreva("Digite o numero de horas trabalhadas no mes: ")
    leia(horastrabalhadas)
   
    salariototal <- valorporhora * horastrabalhadas

    escreva("O total do salario do funcionário no referido mes é: ", salariototal)
    ele n esta dando o resulto final

**Comment by RafaelScbo17** [comment, marked as answer]
No portugol studio, usamos o "= ", e não o <-. o correto seria: salariototal = valorporhora * horastrabalhadas
